src/screens/Authentication/Welcome/Welcome.tsx

```diff
@@ -15,29 +15,37 @@ const WelcomeScreen = ({ navigation }: any) => {
   const recaptchaVerifier = useRef(null);
 
   return (
-    <View style={styles.logoContainer}>
-      <Image source={logo} style={styles.logo} />
-      <View style={styles.container}>
-        <Text style={styles.signInText1}>
+    <View style={styles.container}>
+      <View style={styles.logoContainer}>
+        <Image source={logo} style={styles.logo} />
+      </View>
+      <View style={styles.textContainer}>
+        <Text style={styles.welcomeText}>
           Welcome to the Chinese Newcomers Service Center job portal!
         </Text>
-        <Text style={styles.signInText2}>Let's get your account set up.</Text>
-
-        <Pressable
-          style={styles.signInButton}
-          onPress={() => navigation.navigate('PhoneNumberRegister')}>
-          <Text style={styles.signInText3}> SIGN UP </Text>
-        </Pressable>
+        <Text style={styles.subText}>Let's get your account set up.</Text>
+      </View>
+      <View style={styles.buttonContainer}>
+        <StyledButton
+          text="sign up"
+          onPress={() => navigation.navigate('PhoneNumberRegister')}
+          buttonStyle={{}}
```

Comment:
nit: if there's no `buttonStyle` or `textStyle` then you can remove these attributes here

Reply:
<img width="641" alt="Screen Shot 2022-11-29 at 9 08 59 PM" src="https://user-images.githubusercontent.com/104397558/204712491-bbd7a390-0296-48b0-aa07-2c4514a946c9.png">

it gets a little mad at me if i don't have all the props, should i just ignore it?

Reply:
ok i retract that then listen to the linter